Sources/MapboxNavigation/NavigationViewController.swift

```diff
@@ -462,11 +468,18 @@ open class NavigationViewController: UIViewController, NavigationStatusPresenter
         UNUserNotificationCenter.current().add(notificationRequest, withCompletionHandler: nil)
     }
     
-    public func showStatus(title: String, spinner: Bool, duration: TimeInterval, animated: Bool, interactive: Bool) {
+    public func addNewStatus(status: StatusView.Status) {
```

Comment:
Similarly, this is a backwards-incompatible change, so `showStatus(title:spinner:duration:animated:interactive:)` will have to be brought back as a deprecated method.